File: cs50x/Week-6/sentimental-mario-more/mario.py
from cs50 import get_int


# TODO
def main():
    height = 0
    while True:
        height = get_int("Height: ")
        if height > 0 and height <= 8:
            break
    for i in range(1, height + 1):
        for j in range(1, height + 1):
            if j > (height - i):
                print("#", end="")
            else:
                print(" ", end="")
            if j == height:
                print(" ", "#" * i, end="")
        print()


main()

File: cs50x/Week-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Declaring Variables
    int height;

    // Do-While Loop to keep prompting the user until they provide a valid response
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Nested Loops to create the pyramids
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}